libs/platform/icon-tab-bar/components/icon-tab-bar-tab/icon-tab-bar-tab.component.ts

```diff
@@ -46,48 +41,37 @@ export class IconTabBarTabContentDirective implements HasElementRef {
     templateUrl: './icon-tab-bar-tab.component.html',
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class IconTabBarTabComponent implements TabConfig {
+export class IconTabBarTabComponent implements ReactiveTabConfig {
     /** Tab Label. */
-    @Input()
-    label: Nullable<string>;
+    label = input<Nullable<string>>();
 
     /** Tab text color. */
-    @Input()
-    color: SemanticColor;
+    color = input<SemanticColor>();
 
     /** Tab icon. */
-    @Input()
-    icon: string;
+    icon = input<string>();
 
     /** Tab icon font family. */
-    @Input()
-    iconFont: IconFont = FD_DEFAULT_ICON_FONT_FAMILY;
+    iconFont = input<IconFont>(FD_DEFAULT_ICON_FONT_FAMILY);
 
     /** Tab counter. */
-    @Input()
-    counter: number;
+    counter = input<Nullable<number>>();
 
     /** whether the tab is selected */
-    @Input({ transform: booleanAttribute })
-    active = false;
+    active = input(booleanAttribute(false), { transform: booleanAttribute });
```

Comment:
I don't understand this part. Why not 
`active = input(false,  { transform: booleanAttribute })`

Reply:
i fixed it, i think nx generator made that mistake